command/cp.go

```diff
@@ -777,6 +798,23 @@ func (c Copy) shouldOverride(ctx context.Context, srcurl *url.URL, dsturl *url.U
 	return stickyErr
 }
 
+// shouldCopyObject checks is object should be copied.
+func (c Copy) shouldCopyObject(object *storage.Object, verbose bool) bool {
```

Comment:
Could you add write unit tests to this method?

Reply:
Done. https://github.com/peak/s5cmd/pull/600/commits/873c42b540b90576db907bb53afc00bf8db9789c